library/src/morpheus-master/sass/base/_extendables.scss

```diff
@@ -90,7 +90,7 @@
 }
 
 .select{
-	background-color: $button-background-color;
+	background-color: $background-color;
```

Comment:
are we getting rid of the variable button-background-color now? If yes, let's delete it everywhere

Reply:
No, the variable is still in use elsewhere for the buttons. It had been inappropriately applied to select elements too. I corrected this in this PR.